src/flash_registry_win.cc

```diff
@@ -22,40 +22,76 @@
 
 namespace toit {
 
+static const int ALLOCATION_SIZE = 2 * MB;
+static const int ENCRYPTION_WRITE_SIZE = 16;
+
 const char* FlashRegistry::allocations_memory_ = null;
+static void* allocations_malloced = null;
 
 void FlashRegistry::set_up() {
+  ASSERT(allocations_malloced == null);
+  ASSERT(allocations_memory() == null);
+
+  allocations_malloced = malloc(ALLOCATION_SIZE + FLASH_PAGE_SIZE);
```

Comment:
Check malloc error?

Reply:
Yeah, maybe. In general, we do not necessarily check all mallocs on the host platforms, because we expect to crash there if we run out. A check here would have to be fatal.